bin/generators/__init__.py

```diff
@@ -0,0 +1,71 @@
+import itertools
+
+import lib
+
+
+def word_generator(length_min=7, length_max=15, perms=False):
+    """
+      Generate the words to be used for bruteforcing
+      > :param length_min: minimum length for the word
+      > :param length_max: max length for the word
+      > :param perms: permutations, True or False
+      > :return: a word
+
+      Example:
+      >>> word_generator()
+      aaaaaa
+      aaaaab
+      aaaaac
+      ...
+    """
+    if not perms:
+        chrs = 'abc'
```

Comment:
Yes, my point exactly. Maybe bruteforcing is just a bad idea in general. How many hits do you think you will really get with combinations of only `abc`?